src/views/DashboardView/index.vue

```diff
@@ -182,9 +183,19 @@ export default {
         console.log(`rejoining session: ${path}`)
       } else {
         console.log(`session terminated`)
-        this.$router.push('/')
+        //this.$router.push('/')
+        const user = UserService.getUser()
+        if (!user.isVolunteer) {
+          const url = '/feedback/null/null/null/student/null/null'
+          router.replace(url)
+        }
+        if (user.isVolunteer) {
+          const url = '/feedback/null/null/null/volunteer/null/null'
+          router.replace(url)
+        }
```

Comment:
Necessary to add two ifs back to back as both these conditions ( for student and volunteer ) have to be checked explicitly.

Reply:
The way our database is set up right now, a user is _either_ a student _or_ a volunteer, and cannot be both or neither. Is there a situation where both of these blocks will be executed? If not, then we don't need to have two ifs back to back.

Reply:
The way it's written here is still good, though.

Reply:
Yeah, fair enough. The solution works with if and else. Made the change. Thanks.